pkg/common/proto/agent.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+service AgentService{
+    rpc PerformFantasyActions(FantasyLandscape) returns (FantasySelections) {}
+}
+
+message FantasyLandscape{
+    LeagueSettings settings = 1;
+    FantasyTeam botTeam = 2;
```

Comment:
Depending on the type of league Id want to know about the current lineups of the other fantasy teams in the league too. As well as the fantasy schedule.

Reply:
Good callout. I'll add that to the todo. Current lineups may complicate the 'when to run' the bots (ordering may change) question. Something we should figure out, but will leave to future change